README.md

```diff
@@ -1,23 +1,26 @@
-# Algolia Search for Salesforce B2C Commerce
+# Algolia Search & Discovery for Salesforce B2C Commerce
 
-The Algolia cartridge allows the customer to connect to a hosted search engine that offers full-text, numerical and multifaceted searches, capable of delivering real-time results from the first keystroke. Algolia cartridge replaces the internal Salesforce B2C Commerce search engine with the Algolia search engine. All errors of the service are saved in the log file.
+Algolia’s cartridge unlocks best-in-class Search & Discovery on Salesforce B2C Commerce.
 
-To learn more about the cartridge and how to get started, check out our [documentation](https://www.algolia.com/doc/integration/salesforce-commerce-cloud-b2c/getting-started/introduction/) for more information.
+It can be used for synchronization of product (incl. price and inventory information), category and content data, and includes native storefront cartridges to power Search & Discovery Experiences on both [SFRA (Storefront Reference Architecture)](https://developer.salesforce.com/docs/commerce/sfra/guide/sfra-overview.html) and SiteGenesis. It also is compatible with Headless/Composable storefront integrations, incl. via the [PWA Kit](https://developer.salesforce.com/docs/commerce/pwa-kit-managed-runtime/guide/getting-started.html), accelerated by Algolia’s developer-friendly [InstantSearch](https://www.algolia.com/doc/guides/building-search-ui/what-is-instantsearch/react/), [Autocomplete](https://www.algolia.com/doc/ui-libraries/autocomplete/introduction/what-is-autocomplete) and [Recommend](https://www.algolia.com/doc/ui-libraries/recommend/introduction/getting-started/?client=React) UI libraries. The integration can be used in conjunction with Algolia’s rich suite of Search & Discovery products such as its user-friendly [Merchandising Studio](https://www.algolia.com/doc/guides/managing-results/rules/merchandising-and-promoting/) and [Search Analytics](https://www.algolia.com/doc/guides/search-analytics/overview/), powerful [AI Search](https://www.algolia.com/doc/guides/getting-started/neuralsearch), (Personalization)[https://www.algolia.com/doc/guides/getting-started/neuralsearch/] and [Recommend](https://www.algolia.com/doc/guides/getting-started/neuralsearch/).
```

Comment:
```suggestion
It can be used for synchronization of product (incl. price and inventory information), category and content data, and includes native storefront cartridges to power Search & Discovery Experiences on both [SFRA (Storefront Reference Architecture)](https://developer.salesforce.com/docs/commerce/sfra/guide/sfra-overview.html) and SiteGenesis.
It also is compatible with Headless/Composable storefront integrations, such as the [PWA Kit](https://developer.salesforce.com/docs/commerce/pwa-kit-managed-runtime/guide/getting-started.html), thanks to Algolia's developer-friendly [InstantSearch](https://www.algolia.com/doc/guides/building-search-ui/what-is-instantsearch/react/), [Autocomplete](https://www.algolia.com/doc/ui-libraries/autocomplete/introduction/what-is-autocomplete) and [Recommend](https://www.algolia.com/doc/ui-libraries/recommend/introduction/getting-started/) UI libraries.
The integration can be used in conjunction with Algolia’s rich suite of Search & Discovery products such as its user-friendly [Merchandising Studio](https://www.algolia.com/doc/guides/managing-results/rules/merchandising-and-promoting/) and [Search Analytics](https://www.algolia.com/doc/guides/search-analytics/overview/), powerful [AI Search](https://www.algolia.com/doc/guides/getting-started/neuralsearch), [Personalization](https://www.algolia.com/doc/guides/personalization/what-is-personalization/) and [Recommend](https://www.algolia.com/doc/guides/getting-started/neuralsearch/).
```